universal-login-react/test/core/utils/getMinimalAmountForFiatProvider.test.ts

```diff
@@ -22,6 +23,26 @@ describe('getMinimalAmountForFiatProvider', () => {
       const smallMinimalAmount = '0.0001';
       expect(await getMinimalAmountForFiatProvider(paymentMethod, smallMinimalAmount, tokenPricesService)).to.eq('1');
     });
+
+    it('return rounded provider minimal amount for Wyre', async () => {
+      const bigMinimalAmount = '2.000000156565';
+      const paymentMethod = TopUpProvider.WYRE;
+      expect(await getMinimalAmountForFiatProvider(paymentMethod, bigMinimalAmount, tokenPricesService)).to.eq('2.0001');
+    });
+
+    it('return correct minimal amount for DAI for Wyre', async () => {
+      const bigMinimalAmount = '1.5';
+      const daiTokenDetails = {
+        symbol: 'DAI',
+        name: 'dai',
+        address: '0x9Ad7E60487F3737ed239DAaC172A4a9533Bd9517',
+      } as TokenDetails;
+      const daiPriceInEth = 20;
+      (tokenPricesService.getTokenPriceInEth as any) = () => daiPriceInEth;
+      const expectedMinimalAmount = ValueRounder.ceil(utils.formatEther(utils.parseEther(bigMinimalAmount).mul(daiPriceInEth)));
```

Comment:
Can we add hardcoded number here? If these calculations are wrong then we won't notice, because these are the same as in written function.